content/applications/marketing/events/create_events.rst

```diff
@@ -0,0 +1,427 @@
+communication from running, if it was active or pending.
+
+There is no limit to how many communications can be added in the :guilabel:`Communication` tab of an
+event form.
+
+.. _create_events/event_form/questions:
+
+Questions tab
+-------------
+
+In the :guilabel:`Questions` tab of an event form, users can create brief questionnaires for
+registrants to interact with, and respond to, after they register for the event.
+
+These questions can be focused on gathering basic information about the attendee, learning about
+their preferences, expectations, and other things of that nature. This information can also be used
+to create more detailed reporting metrics, in addition to being utilized to create specific lead
+generation rules.
+
+.. image:: create_events/questions-tab.png
+   :align: center
+   :alt: Typical questions tab on an event form in the Odoo Events application.
+
+.. note::
+   By default, Odoo provides three questions in the :guilabel:`Questions` tab for every event form.
+   The default questions require the registrant(s) to provide their :guilabel:`Name` and
+   :guilabel:`Email`, and make it optional to include their :guilabel:`Phone` number, as well.
+
+   The information gathered from the :guilabel:`Questions` tab can be found on the
+   :guilabel:`Attendees` dashboard, accessible via the :guilabel:`Attendees` smart button. Odoo
+   populates individual records that contain basic information about the registrant(s), as well as
+   their preferences.
+
+To add a question in the :guilabel:`Questions` tab, click :guilabel:`Add a line`. Doing so reveals a
+:guilabel:`Create Question` pop-up window. From here, users can create and configure their question.
+
+.. image:: create_events/create-question-popup.png
+   :align: center
+   :alt: The Create Question pop-up window that appears in the Odoo Events application.
+
+First, enter the question in the field at the top of the form. Then, decide if the question should
+require a :guilabel:`Mandatory Answer` and/or if Odoo should :guilabel:`Ask once per order`, by
+ticking their respective boxes, if desired.
+
+If the :guilabel:`Ask once per order` checkbox is ticked, the question will only be asked once, and
+its value is propogated to every attendee in the order (if multiple tickets are purchased at once).
+If the checkbox is *not* ticked for this setting, Odoo will present the question for every attendee
+that is connected to that reservation/registration.
+
+Next, select a :guilabel:`Question Type` option. The following are the :guilabel:`Question Type`
+options:
+
+- :guilabel:`Selection`: provide answer options to the question for registrants to choose from.
+  Selectable answer options are added in the :guilabel:`Answers` column at the bottom of the pop-up
+  window. This is the only :guilabel:`Question Type` where the :guilabel:`Answers` tab appears.
+  There are no correct or incorrect answers.
```

Comment:
I dont think it is necessary to state this, or that there are no correct or incorrect answers
```suggestion
  Selectable answer options can be managed in the :guilabel:`Answers` column at the bottom of the
  pop-up window.
```